### PR TITLE
Add Environment Info and WebDriverManager task

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: config.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Browser=chrome
+Implicit_wait_time=20
+Explicit_wait_time=30
+Username=jrojas@freeorg01.com
```
Comment (GustavoZHA) [inline config.properties]: Delet this files

File: src/main/java/core/Manager.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+     *
+     * @return an manager entity.
+     */
+    public static Manager getInstance() {
+        if (manager == null) {
+            manager = new Manager();
+        }
+        return manager;
+    }
+
```
Comment (GustavoZHA) [inline src/main/java/core/Manager.java]: Add javadoc

File: src/test/java/salesforce/login/LoginTest.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import salesforce.ui.pages.HomePage;
+
+public class LoginTest extends BaseTest {
+
+    @BeforeClass
+    public void login() {
+        HomePage homePage = loginPage.loginSuccessful("jrojas@freeorg01.com", "fundacionjalaat13");
+        driver.get(EnvironmentVariable.PROPERTIES.getProperty("Endpoint_url"));
+    }
+
```
Comment (GustavoZHA) [inline src/test/java/salesforce/login/LoginTest.java:19]: Delet blank line

Comment (GustavoZHA) [reply src/test/java/salesforce/login/LoginTest.java:19]: and replace by environment variables

File: src/test/java/salesforce/assets/CreateAssetTest.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import salesforce.login.LoginTest;
+
+public class CreateAssetTest extends LoginTest {
+
+    @Test
+    public void test1() {
+        System.out.println("Successful");
+        Assert.assertTrue(assetPage.getCreateAssetBtn() != null);
+    }
+
```
Comment (cristianfchq) [inline src/test/java/salesforce/assets/CreateAssetTest.java:22]: maybe this line should be removed

**Review by cristianfchq — APPROVED** [review]

File: src/main/java/core/Manager.java (hunk 2 above)
Comment (JRojas999) [reply src/main/java/core/Manager.java]: This class was changed.

File: src/test/java/salesforce/login/LoginTest.java (hunk 3 above)
Comment (JRojas999) [reply src/test/java/salesforce/login/LoginTest.java:19]: I did those changes.

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**